Makefile

```diff
@@ -16,6 +16,8 @@ BINARIES=$(addprefix bin/,$(COMMANDS))
 
 GO_LDFLAGS=-ldflags "-X `go list ./version`.Version=$(VERSION)"
 
+DOCKER_BUILDTAGS+="experimental"
```

Comment:
@anusha-ragunathan I was more interested in that for the interim.

Do we have some way to detect if the engine supports plugin API?